src/services/completions.ts

```diff
@@ -4299,6 +4298,34 @@ namespace ts.Completions {
             }
             return undefined;
         }
+

```

Comment:
Intentional?

```suggestion
        }
```

or

```suggestion
        }
        return undefined;
```